src/components/Header/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+import Link from 'next/link';
+
+import IconArrow from 'assets/icons/icon-arrow.svg';
+import IconLogo from 'assets/icons/logo.svg';
+
+const categories = [
+  { name: 'All', href: '/' },
+  { name: 'Web', href: '/tags/web' },
+  { name: 'Mobile', href: '/tags/mobile' },
+  { name: 'Design', href: '/tags/design' },
+  { name: 'Product', href: '/tags/product' },
+];
+
+const Header = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  return (
+    <header className="app-header">
+      <nav
+        className={`app-navigation-bar navbar ${
+          isMenuOpen ? 'app-navigation-bar--menu-open' : ''
+        }`}
+      >
+        <Link href="/">
+          <a
+            href="/"
```

Comment:
```suggestion
```
@hoangmirs I think we can omit `href` from this child as we've already used in the `Link`

Reply:
Yes, we can omit it, but ESLint will raise errors there. 
You can check this issue: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md#case-i-use-nextjs-and-im-getting-this-error-inside-of-links

I went with the solution adding `passHref` attribute to Link components and fixed in https://github.com/nimblehq/til-web/pull/50/commits/2af18151cc87fc2dd5e5af94bd10c14fb14592ed

Reply:
🥲